views/trust-details.html

```diff
@@ -1,5 +1,7 @@
 {% extends "layout.html" %}
 
+{% set title = "Tell us about the trust" %}
```

Comment:
Not sure how this addresses the defect as it suggests that the text 'Historical beneficial owner' should feature in the tab name? Or maybe suggestion in the defect is wrong?

Reply:
![Screenshot 2023-12-20 at 14 28 16 (2)](https://github.com/companieshouse/overseas-entities-web/assets/137518530/7c464765-b7e7-4879-b1ef-da4830bd966e)

This is the page mentioned in the ticket. My assumption is that the suggestion in the defect was possibly incorrect.

Reply:
It ('Historical beneficial owner') was merely an example of how the code looks on another page within the Trusts journey. 

So for this page it should show in the browser tab (and view source) as 
`Tell us about the trust - Register an overseas entity and tell us about its beneficial owners - GOV.UK`